Soccar [UI]/src/nl/soccar/ui/fx/drawable/CarUiFx.java

```diff
@@ -72,9 +78,9 @@ public void draw(GraphicsContext context) {
             // TODO : Change location of sending this message (Ugly implementation for now)

```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1135)